c2corg_api/views/document_schemas.py

```diff
@@ -105,7 +105,9 @@ def get_all_fields(fields, activities, field_list_type):
     """
     fields_list = [
         fields.get(activity).get(field_list_type) for activity in activities
+        if fields.get(activity)
```

Comment:
Why is this new check required?

Reply:
I was getting AttributeError: 'NoneType' object has no attribute 'get', for the paragliding activity

Reply:
Do you still have the test-case that produced the error? I would like to avoid that this fix is hiding another problem.

Reply:
Simply by going to http://0.0.0.0:6553/routes. I think the same happens here http://c2corgv6-demo.gis.internal:6555/routes. I havent got access to this server ATM, so I cant be sure

Reply:
Ok, I see. `paragliding` is not listed in [c2corg_common/fields_route.py](https://github.com/c2corg/v6_common/blob/9a7f0cb814f88a4f7bdbd7443db6cc2e4ada2641/c2corg_common/fields_route.py). But if I remember correctly, there are no paragliding routes, @asaunier?

So, this might be an issue with your test data.

Reply:
The data I use is a dump from june 2017.
I will see with Alex (he isn't working today)

Reply:
@tsauerwein Indeed, there are no paragliding routes, only paragliding outings. Because with a paraglider there is no predefined trajectories, you go up using an hiking/climbing/skitouring/... route and you go down by the airs.
When creating/editing a route you cannot select the paragliding activity (but you can for outings).

Reply:
http://c2corgv6-demo.gis.internal:6555/routes is my instance on the dev server, it's not a good idea to make tests on it since it might be broken because of work in progress, in addition the ES indexing is a bit broken, I have not spend time fixing it yet.

Reply:
Were you able to find out why there was a route with paragliding as an activity?

Reply:
> Were you able to find out why there was a route with paragliding as an activity?

There are probably none. In my dev instance (same data than @julsbreakdown 's?) or in prod:
```
c2corg=# select count(*) from guidebook.routes where 'paragliding' = any(activities);
 count 
-------
     0
(1 row)
```